colibri/src/blockchain/types.rs

```diff
@@ -0,0 +1,47 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum SupportedBlockchain {
```

Comment:
this might be confusing since you don't list all of them. Can you point in the docstring that this isn't the same as rotki since you only use the evm chains? Maybe is worth renaming it to SupportedEvmChains?

Reply:
added a docstring.